packages/core/src/Stores/client-store.js

```diff
@@ -346,6 +346,7 @@ export default class ClientStore extends BaseStore {
         this.accounts[loginid].accepted_bch = 0;
         LocalStore.setObject(storage_key, this.accounts);
         LocalStore.set('active_loginid', loginid);
+        this.syncWithSmartTrader(loginid, toJS(this.accounts));
```

Comment:
Does this single implementation of `syncWithSmartTrader` handle logout or is this strictly meant for account switching?

Reply:
It should handle logout, but you're right, at this point that's not happening. I've added the logic to `cleanUp` in `client-store.js`.